query-helpers/acs.js

```diff
@@ -86,67 +86,67 @@ const acsSQL = (ids, isPrevious = false) => `
+      ELSE (1 / base_sum) * SQRT(POWER("marginOfError", 2) - POWER(sum / base_sum, 2) * POWER(base_margin_of_error, 2))
+    END AS "percentMarginOfError",
+    --- isReliable ---
     CASE
-      WHEN cv < 20 THEN true
+      WHEN "correlationCoefficient" < 20 THEN true
       ELSE false
-    END AS is_reliable
+    END AS "isReliable"
   FROM (
     SELECT
       --- id ---
       ENCODE(CONVERT_TO(variable, 'UTF-8'), 'base64') AS id,
       --- sum ---
-      SUM(e) AS sum,
-      --- m (do not recalculate for non-aggregate selections)---
+      SUM(estimate) AS sum,
+      --- margin_of_error (do not recalculate for non-aggregate selections)---
       --- coalesce null m to 0 ---
       CASE
-        WHEN SUM(m) IS NULL THEN 0
-        WHEN NOT ${isAggregate(ids)} THEN MAX(m)
-        ELSE SQRT(SUM(POWER(m, 2)))
-      END AS m,
-      --- cv (do not recalculate for non-aggregate selections)---
+        WHEN SUM(margin_of_error) IS NULL THEN 0
+        WHEN NOT ${isAggregate(ids)} THEN MAX(margin_of_error)
+        ELSE SQRT(SUM(POWER(margin_of_error, 2)))
+      END AS "marginOfError",
+      --- correlation_coefficient (do not recalculate for non-aggregate selections)---
       CASE
-        WHEN NOT ${isAggregate(ids)} THEN MAX(c)
-        ELSE (((SQRT(SUM(POWER(m, 2))) / ${CV_CONST}) / NULLIF(SUM(e), 0)) * 100)
-      END AS cv,
+        WHEN NOT ${isAggregate(ids)} THEN MAX(correlation_coefficient)
+        ELSE (((SQRT(SUM(POWER(margin_of_error, 2))) / ${CORRELATION_COEFFICIENT_CONST}) / NULLIF(SUM(estimate), 0)) * 100)
+      END AS "correlationCoefficient",
       --- percent (use MAX() as noop agg for non-agg selections; will be recalculated for aggregate selections)--
       --- coalesce null percent to 0 ---
-      COALESCE(MAX(p), 0) AS percent,
-      --- percent_m (use MAX() as noop agg for non-agg selections; will be recalculated for aggregate selections)---
-      --- coalesce null percent_m to 0 ---
-      COALESCE(MAX(z), 0) AS percent_m,
+      COALESCE(MAX(percent), 0) AS percent,
+      --- percent_margin_of_error (use MAX() as noop agg for non-agg selections; will be recalculated for aggregate selections)---
+      --- coalesce null percent_margin_of_error to 0 ---
+      COALESCE(MAX(percent_margin_of_error), 0) AS "percentMarginOfError",
       --- variable ---
       REGEXP_REPLACE(
         LOWER(variable),
         '[^A-Za-z0-9]', '_', 'g'
       ) AS variable,
-      --- variablename ---
-      variablename,
+      --- variableName ---
+      variableName,
       --- base ---
       base,
       --- category ---
       REGEXP_REPLACE(
         LOWER(category),
         '[^A-Za-z0-9]', '_', 'g'
       ) AS category,
-      --- domain ---
+      --- profile ---
       REGEXP_REPLACE(
-        LOWER(domain),
+        LOWER(profile),
         '[^A-Za-z0-9]', '_', 'g'
-      ) AS domain,
+      ) AS profile,
```

Comment:
Actually, why are we changing this to "profile" I thought we talked about this column really being a topic?